### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
## Before submitting checklist

- [ ] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div align="center">
-    <img src="https://raw.githubusercontent.com/neptune-ai/neptune-client/assets/readme/Github-cover.png" width="1500" />
+    <img src="https://neptune.ai/wp-content/uploads/2024/07/Facebook-cover-image.png" width="1500" />
```
Comment (Raalsky) [inline README.md:2]: Commit to orphaned `assets` branch and use github permalink :wink:

Comment (Raalsky) [reply README.md:2]: I mean the imagefile

Comment (SiddhantSadangi) [reply README.md:2]: `assets` is also a protected branch :/

Comment (Raalsky) [reply README.md:2]: So make a small PR :wink:

Comment (SiddhantSadangi) [reply README.md:2]: https://github.com/neptune-ai/neptune-client/pull/1849

**Review by Raalsky — APPROVED** [review]

**Pull request merged**